### PR TITLE
CSS adjustments

### PR DESCRIPTION
**Pull request by aesedepece**

### CODE REVIEW

File: components/Button.vue [hunk 1]
```diff
@@ -46,19 +46,27 @@ export default {
...
   width: fit-content;
   height: fit-content;
   box-sizing: border-box;
+  min-width: 180px;
+  position: relative;
+  text-align: left;
+  letter-spacing: -0.02em;
 
   .arrow {
+    position: absolute;
```
Comment (gabaldon) [inline components/Button.vue:67]: The arrow should be also centered 
Captura de pantalla 2021-04-14 a las 16 14 58

File: components/Hero.vue [hunk 2]
```diff
@@ -58,16 +52,21 @@ export default {
...
-        90deg,
-        $aquamarine-1 0%,
-        $aquamarine-1 43%,
-        $green-2 100%
-      );
+      background: linear-gradient(90deg, $aquamarine-1 43%, $green-2 100%);
       background-clip: text;
       -webkit-text-fill-color: transparent;
     }
+    &::after {
```
Comment (gabaldon) [inline components/Hero.vue:61]: I think it is better if you include the dot inside a span to add a different style. Or make the container bigger so it fits on one line
Captura de pantalla 2021-04-14 a las 16 18 31

File: components/Button.vue (hunk 1 above)
Comment (aesedepece) [reply components/Button.vue:67]: It looks centered in my system 😶
image

File: components/Hero.vue (hunk 2 above)
Comment (aesedepece) [reply components/Hero.vue:61]: Problem is that none of the imported fonts uses a square dot character.

Comment (aesedepece) [reply components/Hero.vue:61]: Let's wait for the Avenir font to be in place and then adjust so it falls in the same line.

File: components/Button.vue (hunk 1 above)
Comment (aesedepece) [reply components/Button.vue:67]: This is the issue preventing the buttons from looking perfect in every browser:
https://bugs.chromium.org/p/chromium/issues/detail?id=1023654

**Comment by aesedepece** [comment]
Rebased :)

**Review by gabaldon — APPROVED** [review]

**Comment by aesedepece** [comment]
Merged in!

**Pull request closed without merging**